### PR TITLE
tests for annulus & hollow_sphere benchmarks

### PR DESCRIPTION
**Pull request by cedrict**

### CODE REVIEW

File: tests/annulus.cc [hunk 1]
```diff
@@ -0,0 +1,562 @@
+#include <aspect/material_model/simple.h>
```
Comment (gassmoeller) [inline tests/annulus.cc]: if this is a copy of the file in benchmarks, please use a `#include ../benchmarks/.../...cc` instead. That way we can test the actual code in benchmarks.

File: tests/hollow_sphere.cc [hunk 2]
```diff
@@ -0,0 +1,696 @@
+#include <aspect/material_model/simple.h>
```
Comment (gassmoeller) [inline tests/hollow_sphere.cc]: same comment as above

**Comment by bangerth** [comment]
Testing also failed -- might have to update the test output

**Comment by tjhei** [comment]
yes, and rebase to the current master to fix the geoid test

File: tests/hollow_sphere.prm [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  set Strategy                           = velocity
+end
+
+
+############### Parameters describing what to do with the solution
+# The final section allows us to choose which postprocessors to
+# run at the end of each time step. 
+
+subsection Postprocess
+  set List of postprocessors = visualization, velocity statistics, HollowSpherePostprocessor, depth average, memory statistics
```
Comment (tjhei) [inline tests/hollow_sphere.prm]: Can you remove the memory postprocessor and update the results for both tests, please? The number is unlikely to be very stable.

**Comment by gassmoeller** [comment]
I think Timo's comment was hidden by other changes, but could you remove the memory statistics postprocessor and update the tests? This number might change.

**Comment by cedrict** [comment]
I did this yesterday evening

**Comment by gassmoeller** [comment]
Sorry, looked at the prm files for the benchmark folder, not the tests. Good to go.

**Pull request merged**